### PR TITLE
Visual indicator on homepage calendar

### PR DESCRIPTION
**Pull request by Minakshi-Verma**
# Description
This feature is intended for our homepage calendar. It adds a visual indicator (a colorful dot) on the dates that has scheduled event on them. Now the user can see the dates that has event on them just by a glance at the calendar. User can click the dates with indicator to check more details about the events scheduled for that particular day.

##Type of change
- [X] New feature (non-breaking change which adds functionality)

## Checklist
- [X] I have performed a self-review of my own code
- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by fireblastdaniel** [comment]
Is it possible to get rid of the dots that appear on the calendar outside of the month's date range? For example, when you look at Feb 2020, there is a green dot in the middle of white space before the 1st that belongs to a january event, but isn't paired with a number you can see on the calendar

**Comment by Minakshi-Verma** [comment]
That's a good suggestion, Daniel. We will look if we can make it fixed. Thanks!

**Comment by Minakshi-Verma** [comment]
Addressed problem resolved! Calendar does not show any dot on blank days now.

File: src/components/events/EventList.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import PropTypes from 'prop-types'
 import EventListCard from './EventListCard'
 import LoadingLogo from '../loading/LoadingLogo'
 import ViewToggle from './ViewToggle'
+// import SelectedRange from '../daypicker/selectedRange'
```
Comment (ryan-hamblin) [inline src/components/events/EventList.jsx:8]: Remove commented-out code. It has no place in master.

**Pull request merged**